.github/workflows/build_and_test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         run: npm ci
 
       - name: Build Web App
-        run: npx gulp build --platform=browser
+        run: npx gulp build --platform=browser --buildMode=debug
```

Comment:
This got really verbose. The platform is now mandatory, so perhaps we should use a positional argument instead.
We can consider having a default build mode too, though in general I like having release be the default to make sure we test the right thing, falling back to debug only when needed. It also reduces the risk of deploying debug code.

Reply:
mm, apologies, I thought you liked verbose e.g. explicit. having platform be a positional arg seems good. a couple questions - 

- where is it best for the defaults to live - in gulp? or in something like package_common/build.sh? 
- i feel like defaults are best picked in the line of development cuz really what you're doing is saving time for devs. my case for making debug the default is that what's protecting us from deploying debug code is the release process. we can also include the buildMode in the artifact name for extra clarity. basically, if we make "release" the default, day-to-day development doesn't get any lift

Reply:
I agree with you about making debug the default, especially if we can see it easily in the file name and version string. (Flutter adds a "Debug" ribbon to the UI)

The question of where this should live is a tricky one. I find it quite helpful to see what each command needs, so the explicit wiring helps in that regard.
I think ideally we add the defaults to the entry points. Maybe the commands that we list in the README.

BTW, being verbose here in the release script is actually good. Let's keep it that way. But let's change buildMode to release though. That's what we really care about, and people may not be testing that while developing.